lib/ecto/schema.ex

```diff
@@ -255,6 +255,8 @@ defmodule Ecto.Schema do
   * `__schema__(:primary_key)` - Returns a list of primary key fields (empty if there is none);
 
   * `__schema__(:fields)` - Returns a list of all non-virtual field names;
+  * `__schema__(:db_fields)` - Returns a list of all field names as stored in db;
+  * `__schema__(:aliases)` - Returns a list of fields with their changed source;
```

Comment:
I disagree with this, because the column name is not an alias for our field, it's the other way around. I would suggest using something like `field_column_name` or `field_source` internally.